requirements.txt

```diff
@@ -0,0 +1,5 @@
+# Make sure clams-python version is explicitly specified, at least the lower bound
+clams-python==1.2.4
+scikit-learn==1.5.0
+lapps==0.0.2
+mmif-python==1.0.16
```

Comment:
mmif-python is a "hard-pinned" dependency of clams-python (https://github.com/clamsproject/clams-python/blob/45ef237c8abc8aa56399db15ae73919c30094c7c/requirements.txt#L1), so it's not safe to have it here again as another hard-pin.